promo/README.md

```diff
@@ -0,0 +1,62 @@
+# 🎉 Promotional material
+
+Open Source Diversity promotional material like logos, banners, stickers and presentation slides. Everyone is welcome to print material and present Open Source Diversity at events – permission is not needed, everyone in the community can help! 😊
+
+
+## 🌼 Logo
+
+![](sticker-45mm.jpg)
+
+The logo is white on `#45b086` green/turquoise. If you need the logo on white, use the inverse. There are more logo versions in the `img` folder.
+
+As typeface we use [Nunito](https://github.com/opensourcediversity/opensourcediversity.org/tree/master/css) in Regular and Bold. (No other weights and no italic styles.)
+
+
+
+## 🖼 Banner
+
+If you write a Tweet, Mastodon toot or other kind of post and need a bigger banner, use this:
+
+![](banner-website.png)
+
+
+
+## ⭕ Stickers
+
+![](sticker-mockup.png)
+
+We usually order [Flyeralarm outdoor stickers](https://www.flyeralarm.com/de/shop/configurator/index/quantity/7983970#159=601&160=602&161=615&162=585), 4.5 cm round. If you want to order some, use the file `sticker-45mm.jpg` for printing.
+
+
+
+## 📺 Presentation
+
+![](banner-presentation.png)
+
+We have the `Open Source Diversity presentation.pdf` which is roughly 20 minutes long, and `Open Source Diversity presentation lightning.pdf` for 5-minute lightning talks.
+
+Anyone is encouraged to give the presentation at a conference! 😊 If you do so, feel free to use these texts for submitting the talk:
+
```

Comment:
Maybe putting the whole part down with the text for the talk in a quote or code container? otherwise is hard to know where it ends.